modules/console/pages/config/security/authorization.adoc

```diff
@@ -229,6 +231,67 @@ authorization:
 
 All APIs must be accessible using a superuser principal. If the service account lacks superuser status, Redpanda Console may not be able to fetch cluster status, access the Admin API, or interact with the Schema Registry.
 
+=== Transform identities with principal mappings
+
+If you use OIDC login, the identity in the JWT token (the `sub` claim by default) must match a `name` in the `roleBindings` configuration. You can use the `principalMapping` configuration option to transform the identity claim into a username format expected by your role bindings.
```

Comment:
WDYT about mentioning that this defaults to `$.sub` which just extracts the `sub` claim as is, without transforming the value. I think this makes the syntax a little clearer than saying "the sub claim by default".